containers/Makefile

```diff
@@ -35,9 +35,13 @@ grpc_clone:
 build-builder: grpc_clone
 	docker build -f docker/server/Dockerfile --tag ${BUILDER_IMAGE_NAME} .
 
-build-builder-client:
+build-builder-client: .BIN_FMT
 	docker build -f docker/client/Dockerfile --tag ${CLIENT_BUILDER_IMAGE_NAME} .
 
+.BIN_FMT:
+	docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+	touch .BIN_FMT
```

Comment:
Restarting the host will keep the `.BIN_FMT` file, but the binfmt entries will be removed.
In my machine it takes less than a second to run this docker command, I don't see a reason to enforce it to run only once.